spatialdata/_core/_spatialdata.py

```diff
@@ -200,6 +244,20 @@ def is_backed(self) -> bool:
 
     # TODO: from a commennt from Giovanni: consolite somewhere in a future PR (luca: also _init_add_element could be cleaned)
     def _get_group_for_element(self, name: str, element_type: str) -> zarr.Group:
+        """Get the group for an elemnt, creates a new one if the element doesn't exist
+
+        Parameters
+        ----------
+        name : str
```

Comment:
in general, parameters should not have types as they are already present in signature. so this (and everything else) should just be `name`